app/src/org/commcare/activities/FormEntryActivityUIController.java

```diff
@@ -372,6 +380,11 @@ protected void showNextView() {
     }
 
     private void showNextView(boolean resuming) {
+        if (shouldIgnoreNavigationAction()) {
+            isAnimatingSwipe = false;
```

Comment:
I had to add this to fix a bug next() and showNextView() could accidentally end up leaving the animating flag to true.